### PR TITLE
V3 docs upgrade

### PR DESCRIPTION
**Pull request by obrienben**
Relevant documentation updated for v3 - work in progress

### CODE REVIEW

**Comment by obrienben** [comment]
@hannakoppelaar could you please have a look at the changes so far?

File: webcurator-docs/guides/system-administrator-guide.rst [hunk 1]
```diff
@@ -73,29 +63,27 @@ Other versions of the required products may be compatible with the Web
...
 -------------------
 
 The following platforms have been used during the development of the Web
 Curator Tool:
 
--  Sun Solaris 10
-
 -  Red Hat Linux EL3.
 
 -  Ubuntu GNU/Linux 16.04 LTS
```
Comment (hannakoppelaar) [inline webcurator-docs/guides/system-administrator-guide.rst:76]: And Ubuntu 18.04 and Linux Mint 19.1

File: webcurator-docs/guides/system-administrator-guide.rst [hunk 2]
```diff
@@ -54,16 +51,9 @@ Curator Tool:
 
 -  Java 1.8 JDK or above (64bit recommended)
```
Comment (hannakoppelaar) [inline webcurator-docs/guides/system-administrator-guide.rst:52]: I guess *only* jdk 1.8 for now?

File: webcurator-docs/guides/system-administrator-guide.rst [hunk 3]
```diff
@@ -73,29 +63,27 @@ Other versions of the required products may be compatible with the Web
...
 ---------------
 
 The following platforms were used during the Development of the Web
 Curator tool but are not explicitly supported:
 
--  Windows 2000, Windows XP Pro, Windows Server 2003
+-  Sun Solaris 10
 
 Optional prerequisites
 ----------------------
```
Comment (hannakoppelaar) [inline webcurator-docs/guides/system-administrator-guide.rst:89]: We now also need gradle to build from source. I'm building with version 4.4.1, which is really old, everything newer than that should be fine.

File: webcurator-docs/guides/system-administrator-guide.rst [hunk 4]
```diff
@@ -241,241 +215,198 @@ data for the new QA module for each agency, and should be run* **once all
...
-    port number.
+   - install_maven_dependencies.bat
+   - install_maven_dependencies.sh
 
-    For Tomcat, this is done by adding the following to your
-    `$TOMCAT_HOME/bin/catalina.sh script`::
+- Navigate back to the root *webcurator* directory, and build the project using Gradle::
 
-        JAVA_OPTS=-Dcom.sun.management.jmxremote.port=9004
+   gradle clean bootWar --project-prop include<database-type>=true
```
Comment (hannakoppelaar) [inline webcurator-docs/guides/system-administrator-guide.rst:260]: It's no longer necessary to include database type when building: all JDBC drivers are included in the binary.

The bootWar target doesn't build everything, currently. In one of the other pending PRs (https://github.com/WebCuratorTool/webcurator/pull/6) I've changed the top-level build command to "gradle clean install".

File: webcurator-docs/guides/system-administrator-guide.rst [hunk 5]
```diff
@@ -241,241 +215,198 @@ data for the new QA module for each agency, and should be run* **once all
...
+   in the foreground, simply use *Ctrl+c*.
 
-    #Hibernate Settings
-
-    hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
-    hibernate.default_schema=DB_WCT
+-  Before logging into WCT, *ensuring all components are shutdown*, modify the
+   configuration files in the following steps.
 
+Configure the Database Connection
```
Comment (hannakoppelaar) [inline webcurator-docs/guides/system-administrator-guide.rst:319]: An easier way to configure the components would be to have users add a local application.properties file in the working directory of the java -jar command (e.g. application-local+mysql.properties for webapp, if we're using MySQL). This config file would only need to contain the properties from the built-in application.properties and application-local+mysql.properties that they want to override. Then the war and jar files wouldn't have to be altered (except if you want to change the default Heritrix profile).

In the case of webapp you can select the active profile with the command line parameter --spring.profiles.active=local+mysql. The other components don't have separate profiles, but just a single application-local.properties (that we can override in the same way as application-local+mysql.properties).

File: webcurator-docs/guides/system-administrator-guide.rst (hunk 1 above)
Comment (obrienben) [reply webcurator-docs/guides/system-administrator-guide.rst:76]: Will add

File: webcurator-docs/guides/system-administrator-guide.rst (hunk 4 above)
Comment (obrienben) [reply webcurator-docs/guides/system-administrator-guide.rst:260]: I've updated to reflect this

File: webcurator-docs/guides/system-administrator-guide.rst (hunk 2 above)
Comment (obrienben) [reply webcurator-docs/guides/system-administrator-guide.rst:52]: PR submitted for >1.8 compatibility

**Comment by obrienben** [comment]
Superseded by _v3.0/update-docs_ branch

**Pull request closed without merging**